server/models/Notification.ts

```diff
@@ -182,6 +184,40 @@ export class Notification extends Model<InferAttributes<Notification>, InferCrea
     return notifications.map(us => us.UserId);
   }
 
+  static async getUnsubscribers(_where: {
+    type?: ActivityClasses | ActivityTypes;
+    CollectiveId?: number | number[];
+    UserId?: number | number[];
+  }) {
+    debug('getUnsubscribers', { _where });
+    const getUsers = notifications => notifications.map(notification => notification.User);
+
+    const include = [{ model: models.User }] as any;
```

Comment:
With:

```es6
import { Includeable } from 'sequelize';
```

```suggestion
    const include = [{ model: models.User }] as Includeable[];
```

Also curious about why we need to cast the `where` below to `any`, there's probably a more elegant solution.

Not a blocker if this is time-consuming.